content/using-atom/sections/github-package.md

```diff
@@ -113,8 +124,29 @@ Sometimes there can be conflicts when trying to merge. Files that have merge con
 ![Resolve conflicts](../../images/github-resolve-conflicts.png "Resolve conflicts")
 
 
+#### Create a Pull Request
+
+When your changes are ready to be reviewed by your team members, open the "GitHub" panel <kbd>Ctrl+8</kbd> and click on **Open new pull request**. It will open the browser where you can continue creating a pull request. If commits haven't been pushed or the branch isn't published yet, the GitHub package will do that automatically for you.
+
+![Create a Pull Request](../../images/github-create-a-pull-request.png "Create a Pull Request")
+
+
 #### View Pull Requests
 
-If there is a **Pull Request** for your current branch, you can see its status in the "GitHub" panel. Click on **Conversation** to see the timeline.
+Once the pull request is created, it will appear under **Current pull request** at the top of the panel. Underneath is a list of **Open pull requests**. It lets you quickly find a pull request by avatar, title or PR number. It also lets you keep an eye on the CI status. Clicking on a pull request in the list opens a center pane with more details, the timeline and conversations.
 
 ![View Pull Requests](../../images/github-view-pull-requests.png "View Pull Requests")
+
+
+#### Checkout a Pull Request
+
+To test a pull request locally, click on the **Checkout** button. It will automatically create a local branch and pull all the changes. If you would like to contribute to that pull request, start making changes, commit and push. Your contribution is now part of that pull request.
```

Comment:
Let's add a step for how to get to the button again, just in case someone jumps here directly instead of reading sequentially from the View Pull Requests section:

> To test a pull request locally, open it in the workspace center by clicking on the pull request in the **open pull requests** list from the GitHub tab, then click on the **Checkout** button.